### PR TITLE
Add note to readme about memory limit and error code 137

### PR DESCRIPTION
**Pull request by mikedikan**

### CODE REVIEW

**Comment by mikedikan** [comment]
FYI @edx/docker-devstack-working-group

File: README.rst [hunk 1]
```diff
@@ -543,6 +543,20 @@ up the provisioning process on Mac), so you can try the following:
...
+~~~~~~~~~~~~
+
+While provisioning, some have seen the following error:
+
+.. code:: sh
+
+   ...
+   Build failed running pavelib.assets.update_assets: Subprocess return code: 137
+
+This is an indication that your docker process ran out of memory during execution.
```
Comment (robrap) [inline README.rst]: I'd note that return code 137 basically means some went majorly wrong and died.  We have found that this often can be fixed by adjusting memory, but if that doesn't work, more sleuthing will be required.

Comment (mikedikan) [reply README.rst]: comment (I believe) addressed - please take another look :)

**Review by macdiesel — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -543,6 +543,21 @@ up the provisioning process on Mac), so you can try the following:
...
+~~~~~~~~~~~~
+
+While provisioning, some have seen the following error:
+
+.. code:: sh
+
+   ...
+   Build failed running pavelib.assets.update_assets: Subprocess return code: 137
+
+This is an indication that your docker process died during execution.  Most likely,
```
Comment (robrap) [inline README.rst:556]: Nit: What does it actually kill?  I'm unclear if it is the original docker process that dies, or just the additional command you ran, etc.?

File: README.rst [hunk 3]
```diff
@@ -543,6 +543,21 @@ up the provisioning process on Mac), so you can try the following:
...
+this is due to running out of memory (this has historically been the case at edX).
+If you are set to 2GB (docker for mac default) up it to 4GB (the edX recommendation).
```
Comment (robrap) [inline README.rst]: Some minor changes I would make are detailed here:
```
... Most likely, this is due to running out of memory. If you are set to 2GB (docker for mac default), increase it to 4GB (the current recommendation). If you are set to 4GB, try 6GB.
```

**Review by robrap — APPROVED** [review]
I recommended some minor changes, but this would get the job done as-is as well.  Thanks for creating the PR!

File: README.rst (hunk 2 above)
Comment (mikedikan) [reply README.rst:556]: The entire process (make provision) dies, resulting in a exit in the middle of provisioning (very annoying, as we need to restart provisioning)  Should I try to make that more clear?  Suggested language would be helpful if so.

File: README.rst (hunk 3 above)
Comment (mikedikan) [reply README.rst]: Sounds like you want to strip out the personality here 😝

Comment (robrap) [reply README.rst]: I like the personality, but am concerned for people where english might be their second language.  I also removed references to "edX" that weren't needed, or could have been "Open edX", yadda yadda yadda.

File: README.rst (hunk 2 above)
Comment (robrap) [reply README.rst:556]: Just wondering.  I don't think it needs any changes.  Someone will know what it means when they see the error. :)

Comment (staubina) [reply README.rst:556]: @robrap You can get this error in any container as a note.  I had this error in my MySql container when I ran into this issue.

**Review by staubina — APPROVED** [review]

**Pull request merged**